demo/demo.js

```diff
@@ -68,29 +81,50 @@ export function init(formDefaults, render) {
 	return syncFormData;
 }
 
+/**
+ * Translates the form/stored data into the parameters expected by index.js
+ * @param {DemoForm} formData
+ * @returns {RevSDKDemoSettings}
+ */
 function getConfig(formData) {
 	const {
 		config,
 		tokenValue,
 		tokenType,
 		tokenIssuer,
 		sourceUrl,
-		...data
+		embedType,
+		videoId,
+		webcastId,
+		baseUrl
 	} = formData
 
-	return {
-		...data,
+	/** @type {RevSDKDemoSettings} */
+	const data = {
+		baseUrl,
 		config: {
+			// include any passed in config values (hideControls, accentColor, etc)
 			...tryParse(config),
 			token: tokenValue && {
 				type: tokenType,
 				value: tokenValue,
 				issuer: tokenIssuer
 			}
-		},
+		}
 	};
+
+	// read the form radio toggle to only return expected ID
+	if (embedType === 'vod') {
+		data.videoId = videoId;
+	} else {
+		data.webcastId = webcastId;
+	}
+
+	return data;
 }
```

Comment:
in some circumstances it was passing both videoId and webcastId when parsing the form. This makes the determination of what to pass explicit rather than relying on form.elements parsing. The previous code worked, this just may make it clearer